routes/api/users.js

```diff
@@ -69,7 +70,12 @@ router.post("/users/login", function(req, res, next) {
     })(req, res, next);
 });
 
-router.post("/users", function(req, res, next) {
+
+
+router.post("/users", validate.validateSignup, function(req, res, next) {
```

Comment:
A better naming scheme for `validate.validateSignup` will be `validate.signup` because the `validate` object has qualified the `signup`. There is no need for repeating `validate`.